roles/awsenv/tasks/network.yml

```diff
@@ -49,15 +50,15 @@
 
   - name: Set value of new gateway id for route table
     set_fact:
-      nat_gw_id: "{{ nat_gw.nat_gateway.nat_gateway_id }}"
+      nat_gw_id: "{{ nat_gw.nat_gateway_id }}"
```

Comment:
L53: I see you dropped nat_gw.**nat_gateway** .nat_gateway_id.
I don't see nat_gw listed as a change in here, so I presume this change is due:

[ ] a typo
[ ] fixing a bug, this never worked
[ ] lolipops